packages/airnode-validator/src/config/config.test.ts

```diff
@@ -2,6 +2,7 @@ import { readFileSync } from 'fs';
 import { join } from 'path';
 import { ZodError } from 'zod';
 import zip from 'lodash/zip';
+import references from '@api3/airnode-protocol/dist/deployments/references.json';
```

Comment:
Adding `dist/` was necessary for the `airnode-client` to find the file given the `airnode-protocol` outdir specified in `tsconfig.json` is `dist`:

https://github.com/api3dao/airnode/blob/c43779b969fff7bf6ab7213fd7a212dbcfbf156f/packages/airnode-protocol/src/tsconfig.json#L8C7-L10

but I wonder if there is a cleaner approach to export?

Reply:
Not sure what "Adding dist/" you mean.

Reply:
Ignore- I fixed this import by exporting references from airnode-protocol